gthockey-iOS/gthockey-iOS/JSONParser.swift

```diff
@@ -12,7 +12,7 @@ import AlamofireImage
 import SwiftyJSON
 
 class JSONParser {
-
+    
```

Comment:
Get rid of this spacing. To fix this in Xcode settings: Xcode -> Preferences -> Text Editing. Make sure both of these are checked off to eliminate the extra whitespace.
<img width="375" alt="66278674-608c3900-e879-11e9-88ce-459fdab50262" src="https://user-images.githubusercontent.com/13947356/66613989-52019280-eb95-11e9-8a54-da227f373d95.png">